war/src/main/webapp/help/system-config/globalEnvironmentVariables.html

```diff
@@ -1,4 +1,4 @@
 <div>
-    These key-value pairs apply for every build on every node. They can be used in Jenkins'
```

Comment:
this would be an incorrect change.

Without entering the debate of if it should be Jenkins' or Jenkins's (the former is more commonly used at least in Brittain) the overriding rule is "just use the same everywhere" - given there is plently of other useage of `Jenkins'` in the code (and documentation) this breaks the "use the same everywhere" rule.